daemon/daemon_unix_test.go

```diff
@@ -376,3 +379,75 @@ func sysInfo(t *testing.T, opts ...func(*sysinfo.SysInfo)) sysinfo.SysInfo {
 	}
 	return si
 }
+
+func TestGetBlkioWeightDevices(t *testing.T) {
```

Comment:
`mknod` requires root, so this test case should have something like
```go
        if os.Getuid() != 0 {
                t.Skip("root required") // for mknod
        }
```
at the very beginning (so if someone runs `go test` as non-root, there won't be a fake failure).

Reply:
Also, make sense to use
```go
t.Parallel()
```